converter/Gopkg.toml

```diff
@@ -26,7 +26,7 @@
 
 [[constraint]]
   name = "github.com/activecm/rita"
-  version = "^1.0.3"
+  version = "^2.0.0"
```

Comment:
This should have been updated earlier, but it the change was needed now as we have to get the parsetypes.Freq object out of RITA to get the needed MongoDB indexes for the freqConn collection